Libs/MRML/DisplayableManager/vtkMRMLSliceIntersectionWidget.cxx

```diff
@@ -1392,6 +1411,28 @@ bool vtkMRMLSliceIntersectionWidget::ProcessZoomSlice(vtkMRMLInteractionEventDat
   return true;
 }
 
+
+// TODO: by simzhangbest mail:dg20330034@smail.nju.edu.cn 
+// mouse left-click-drag shit between pics not smooth enough, may someone modify this
+bool vtkMRMLSliceIntersectionWidget::ProcessIncDecSlice(vtkMRMLInteractionEventData* eventData)
+{
+    // Please adjust this parameter to tune the Inc&Dec speed.
+    const double CONF = 0.01;
+
+    const int* eventPosition = eventData->GetDisplayPosition();
+    const int* windowSize = this->GetRenderer()->GetRenderWindow()->GetSize();
+
+    // modify by simzhang 0809
+    int deltaY = eventPosition[1] - this->PreviousEventPosition[1];
+
+    this->MoveSlice(deltaY * CONF);
+
+    // if (deltaY > 0) this->IncrementSlice();
+    // else this->DecrementSlice();
```

Comment:
We don't like to leave dead code in the source so this should be removed once the functionality is in place.